README.md

```diff
@@ -1 +1,28 @@
-# Typing-test-project
\ No newline at end of file
+# TypingTestProject
```

Comment:
Może warto by było uzupełnić readme o opis projektu. W końcu ma to być wizytówka